ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java

```diff
@@ -705,6 +718,39 @@ public BreakpointRenderer getBreakpointRenderer() {
   }
 
   @Override
+  public Promise<VcsChangeMarkerRender> getOrCreateVcsChangeMarkerRender() {
```

Comment:
I disagree to move the method from here. It is placed here to be able to use it for change marker implementation of other VCS.